ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -458,7 +464,8 @@ func (s *Service) buildCreateServiceInput(serviceName, taskDefName string) (*ecs
 	}
 
 	createServiceInput := &ecs.CreateServiceInput{
-		DesiredCount:            aws.Int64(0),            // Required
+		DesiredCount:            aws.Int64(0), // Required unless DAEMON schedulingStrategy
+		SchedulingStrategy:      aws.String("REPLICA"),
```

Comment:
Since `REPLICA` is the default, let's not set anything here and only set `SchedulingStrategy` explicitly once, if the `schedulingStrategy` flag is set & valid.

Reply:
This should be addressed with https://github.com/aws/amazon-ecs-cli/pull/685/commits/a78de9eca550319e3bb5962c4abb777bc4401b0b.